### PR TITLE
Read open office file instead of tsv for multiline support

### PR DESCRIPTION
**Pull request by possan**
This change reads ODS (open office) files from google instead of tsv/csv, which of none seems to support linebreaks, this one will support having markdown in the translation document.

You need to update your env variables to point to this instead:
`TRANSLATIONS_URL="https://docs.google.com/spreadsheets/d/1cL9kxvUgs9Bst2ZiAE3UDly6Azp03iokgtyfXP9fhjk/pub?gid=1904984372&single=true&output=ods"`

Adds node dependencies :/

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "semi": false,
+  "singleQuote": false
```
Comment (mikabytes) [inline .prettierrc:3]: This should probably be "backtick"

Comment (mikabytes) [reply .prettierrc:3]: https://eslint.org/docs/rules/quotes

**Review by mikabytes — CHANGES_REQUESTED** [review]
Kudos for adding eslintrc. 

Looks like ODS is a bit trickier to read. But I'm sure it's a good quality increase :+1:

File: .prettierrc (hunk 1 above)
Comment (possan) [reply .prettierrc:3]: Seems like prettier will not change between strings and template strings, but eslint will, that's why it's only in eslintrc :/

https://github.com/prettier/prettier/issues/54#issuecomment-283439626
https://github.com/prettier/prettier/issues/2996#issuecomment-335223208

Comment (mikabytes) [reply .prettierrc:3]: Ah, yes, I got confused here. Thought I was looking at the eslintrc file. 

Sadly, Prettier is ignoring them. I have a plugin for eslint that does the conversion for me that works really well. Would have preferred this to be built-in for Prettier.

**Review by mikabytes — APPROVED** [review]

**Pull request merged**